index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <metaname=“viewport” content=“width=device-width, initial-scale=1”>
+    <title>Aloha Apparel Co.</title>
+    <link rel="stylesheet" href="lib/flickity/flickity.min.css">
+    <link rel="stylesheet" href="CSS/normalize.css">
+    <link rel="stylesheet" href="CSS/style.css">
+    <link rel="stylesheet" href="font-awesome/css/font-awesome.min.css">
+  </head>
+
+  <body>
+    <header class="top-bar">
+      <img class="logo" src="./images/aloha-logo.svg" alt="">
+
+      <ul class="navigation">
+          <li><a href="#about">about</a></li>
+          <li><a href="#shop">shop</a></li>
+          <li><a href="#featured">featured</a></li>
+          <li><a href="#updates">updates</a></li>
+        </ul>
+       
+   
+      <img class="cart" src="./images/cart-icon.svg" alt="Shopping Cart">
+    </header>
+        
+      <div class="banner">
```

Comment:
Use section elements (rather than divs) for all of the main content chunks on the site.